packages/apollo-gateway/src/loadServicesFromStorage.ts

```diff
@@ -38,12 +38,12 @@ const urlFromEnvOrDefault = (envKey: string, fallback: string) =>
 // Generate and cache our desired operation manifest URL.
 const urlPartialSchemaBase = urlFromEnvOrDefault(
   envOverridePartialSchemaBaseUrl,
-  'https://storage.googleapis.com/engine-partial-schema-prod/',
+  'https://managed-federation.apollographql.com/',
 );
 
 const urlStorageSecretBase: string = urlFromEnvOrDefault(
   envOverrideStorageSecretBaseUrl,
-  'https://storage.googleapis.com/engine-partial-schema-prod/',
+  'https://managed-federation.apollographql.com/',
```

Comment:
Can we use two different URLs here, even if for now they are backed by the same bucket and CDN?

Reply:
@glasser yes, pick them please.